discuss/advanced.rst

```diff
@@ -6,4 +6,5 @@ Advanced
    :titlesonly:
    :glob:
 
-   advanced/emailtemplate
\ No newline at end of file
+   advanced/email_servers
+   advanced/emailtemplate
```

Comment:
```suggestion
   advanced/email_template
```
It's very recent so no need to add a redirection rule.

Reply:
I know. It is a new doc. I did not create a redirection rule. 
Thanks!